examples/tutorial/package.json

```diff
@@ -1,10 +1,12 @@
 {
+  "private": true,
+  "type": "module",
   "scripts": {
     "build": "sass --load-path=node_modules main.scss css/main.css"
   },
   "dependencies": {
-    "@minvws/manon": "18.0.0-alpha.3",
-    "@minvws/manon-themes": "18.0.0-alpha.3"
+    "@minvws/manon": "workspace:*",
+    "@minvws/manon-themes": "workspace:*"
```

Comment:
Perhaps forgotten to revert. This should resemble the tutorial text. Since someone doing the tutorial will not have the repository cloned but will use manon as a package.